crates/ev-prover/src/lib.rs

```diff
@@ -20,6 +20,8 @@ use rsp_rpc_db::RpcDb;
 use std::{fs, sync::Arc};
 use tracing::debug;
 
+pub const ISM_ID: &str = "0x726f757465725f69736d000000000000000000000000002a0000000000000001";
```

Comment:
In the range prover PR the other day I added an ism_id to the client. Which is configured as the "default ism" of the client. This is done via env var since the client is constructed with `from_env`. 

See: https://github.com/celestiaorg/celestia-zkevm-hl-testnet/blob/main/crates/celestia-grpc-client/src/client.rs#L66-L69

And: https://github.com/celestiaorg/celestia-zkevm-hl-testnet/blob/main/.env.example#L16

This should be able to be removed in favour of that API on the client.
Maybe you can address this one before merge.